### DISCUSSION TITLE
New Update on C210 camera (1.4.3) seems to have fixed the integration issue

### DISCUSSION THREAD

**Discussion by AndyRockall** [JurajNyiri/HomeAssistant-Tapo-Control#708] [General]
I just updated two of my cameras to the new version 1.4.3 firmware and can now use them with integration again.
All functions seem to be working OK.
Just waiting for the other cameras to get the update!

**Comment by J-C-P** [comment]
High
I have a few C210 (v2) but they do not offer to update the firmware. Is there anything I can do to speed it up ?
Thanks for your feedback, by the way? It gives hope :-)
Have a good day!

**Reply by AndyRockall** [reply]
Sorry 
I forgot to update my post.
I had to downgrade them to ver. 1.3.7 again as they stopped working again.

**Reply by J-C-P** [reply]
oh :-(
Thanks for the update
